es6/server/controllers/question.js

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs';
+
+export const fetchQuestions = () => {
+	try {
+		const questionDatabase = fs.readFileSync('server/db/questions.json');
+		return JSON.parse(questionDatabase);
+	} catch (error) {
+		return [];
+	}
+};
+
+const saveQuestion = questions => fs.writeFileSync('server/db/questions.json', JSON.stringify(questions, undefined, 2));
+const allQuestions = fetchQuestions();
```

Comment:
Is it possible to have only the class in this file? then have the `fetchQuestions()` and `saveQuestions()` in a folder `Utils`?

Reply:
@Noblemajesty Ok will do.